web/src/app/services/data-store/data-store.service.ts

```diff
@@ -290,19 +327,10 @@ export class DataStoreService {
     return await Promise.all(querySnapshot.docs.map(doc => doc.ref.delete()));
   }
 
-  private async deleteAllLocationsOfInterestInJob(
+  async deleteLocationOfInterest(
     surveyId: string,
-    jobId: string
-  ) {
-    const lois = this.db.collection(
-      `${SURVEYS_COLLECTION_NAME}/${surveyId}/lois`,
-      ref => ref.where(l.jobId, '==', jobId)
-    );
-    const querySnapshot = await firstValueFrom(lois.get());
-    return await Promise.all(querySnapshot.docs.map(doc => doc.ref.delete()));
-  }
-
-  async deleteLocationOfInterest(surveyId: string, loiId: string) {
+    loiId: string
+  ): Promise<void> {
     await this.deleteAllSubmissionsInLocationOfInterest(surveyId, loiId);
```

Comment:
This longer name doesn't help readability imo.. could just be `deleteSubmissionsByLoiId` or `deleteSubmissionsForLoi`